app/models/order.rb

```diff
@@ -0,0 +1,14 @@
+class Order < ApplicationRecord
+  belongs_to :user
+  has_many :order_items, dependent: :destroy
+
+  scope :default_order, -> { order(created_at: :desc) }
+
+  def total_price
+    total_price = 0
+    order_items.each do |order_item|
+      total_price += order_item.price
+    end
+    total_price
```

Comment:
合計、平均、最大、最小みたいなロジックはrailsのメソッドになってることが多いから、自分で計算し始めたら疑問をもったほうがいいね。
```suggestion
    order_items.sum(&:price)
```